tox.ini

```
@@ -12,31 +12,37 @@ extend-ignore=E203
 max-line-length=88
 per-file-ignores=__init__.py:F401
 
+[blocklint]
+max_issue_threshold=1
+
 [testenv]
 usedevelop=True
 install_command=python -m pip install {opts} {packages}
 commands=
-    core: pytest {posargs:tests}
-    docs: make check-docs
+    core: pytest {posargs:tests/core}
```

I think this should still be 
```suggestion
    core: pytest {posargs:tests}
```

---

Yep. The tests should probably be refactored for consistency at some point to avoid a pretty easy-to-make mistake when applying the template that can lead to a false positive of tests, missing 99.9% of them.